packages/react-ui-ag/src/Modals/EmailModal.js

```diff
@@ -0,0 +1,69 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import { Modal, Form, Field, Button } from '@rentpath/react-ui-core'
+
+@themed(/^EmailModal/, { pure: true })
+export default class EmailModal extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    header: PropTypes.node,
+    label: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+    footer: PropTypes.node,
+    onSubmit: PropTypes.func,
+    error: PropTypes.node,
+    email: PropTypes.string,
+    field: PropTypes.object,
+    submitButton: PropTypes.object,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const {
+      className,
+      theme,
+      header,
+      label,
+      footer,
+      onSubmit,
+      error,
+      email,
+      field,
+      submitButton,
+      ...props
+    } = this.props
+
+    return (
+      <Modal
+        className={classnames(
+          className,
+          theme.EmailModal,
+          error && theme['EmailModal-error']
+        )}
+        data-tid="email-modal"
+        {...props}
+      >
+        {header && <div className={theme.EmailModal_Header} data-tid="header">{header}</div>}
+        <Form onSubmit={this.props.onSubmit} serialize>
+          <Field
+            name="email"
+            label={label}
+            defaultValue={email}
+            key={email}
+            {...field}
+          />
+          {error && <span className={theme.EmailModal_ErrorMessage} data-tid="error">{error}</span>}
+          <div className={theme.EmailModal_Submit}>
+            <Button type="submit" name="submit" {...submitButton}>Submit</Button>
```

Comment:
If `submitButton` is passing in Button props, perhaps rename `submitButtonProps` for more clarity.

Consider adding a prop 'submitButtonText' and default to string 'Submit'.